modular_zubbers/code/datums/armor_overrides.dm

```diff
@@ -0,0 +1,29 @@
+// Buffs security modsuit armor
+/datum/mod_theme/security
+	armor_type = /datum/armor/mod_theme_security
+
+/datum/armor/mod_theme_security
+	melee = 40
+	bullet = 30
+	laser = 30
+	energy = 40
+	bomb = 50
+	bio = 100
+	fire = 100
+	acid = 100
+	wound = 10
+
+// Blueshield Armor
+/datum/mod_theme/blueshield
+	armor_type = /datum/armor/mod_theme_blueshield
+
+/datum/armor/mod_theme_blueshield // Not really great for fighting.
+	melee = 30
```

Comment:
```suggestion
	melee = 40
```
we can afford to have it at 40 while being worse for  ranged combat (ie laser/bullet)

Reply:
Agreed